certeasy_api_tests/services/issue_personal_certificate/generate_json.py

```diff
@@ -0,0 +1,89 @@
+import json
+import random
+import string
+from datetime import datetime, timedelta
+
+from certeasy_api_tests.services.issue_tls_server_certificate.generate_json import generate_validity, generate_address, \
+    generate_key_strength, generate_name
+import random
+
+
+def generate_email(username, surname):
+    domains = ["example.com", "test.com", "sample.com", "domain.com"]  # Add more domains as needed
+    domain = random.choice(domains)
+    email = username + surname + "@" + domain
+    return email
+
+
+def generate_user_telephone(country_codes):
+    country_code = random.choice(country_codes)
+
+    if country_code == "Mozambique":
+        # Mozambique telephone numbers have the format +2588XXXXXXX
+        number = "+2588" + "".join([str(random.randint(0, 9)) for _ in range(7)])
+    elif country_code == "South Africa":
+        # South Africa telephone numbers have the format +2783XXXXXXX
+        number = "+2783" + "".join([str(random.randint(0, 9)) for _ in range(7)])
+    elif country_code == "Spain":
+        # Spain telephone numbers have the format +34XXXXXXXXX
+        number = "+34" + "".join([str(random.randint(0, 9)) for _ in range(9)])
+    elif country_code == "EUA":
+        # USA telephone numbers have the format +1-XXX-XXX-XXXX
+        number = "+1-" + "".join([str(random.randint(0, 9)) for _ in range(3)]) + "-" + "".join(
+            [str(random.randint(0, 9)) for _ in range(3)]) + "-" + "".join(
+            [str(random.randint(0, 9)) for _ in range(4)])
+    else:
+        number = "Country code not recognized."
+
+    return number
```

Comment:
- [x] This function is being repeated. 
If possible could be defined in one place only